exopy_hqc_legacy/instruments/drivers/visa/agilent_pna.py

```diff
@@ -885,7 +911,23 @@ def data_format(self, value):
         """
         self.write('FORMAT:DATA {}'.format(value))
         result = self.query('FORMAT:DATA?')
+        
+        self.write('FORMat:BORDer SWAPped')
 
         if result.lower() != value.lower()[:len(result)]:
+            print(result)
             raise InstrIOError(cleandoc('''PNA did not set correctly the
                 data format'''))
+
+    @instrument_property
+    @secure_communication()
+    def data_endianess(self):
```

Comment:
Not sure what the point is since you force the value above anyway. This does not present any true interest to the user and is just a communication details, so I would hide it.